### PR TITLE
Add data table of hours spent on project detail page.

### PR DESCRIPTION
**Pull request by jmcarp**
It looks kind of like this:
screen shot 2015-07-17 at 5 51 04 pm
Just imagine it with more users and reporting periods. @batemapf is this what you had in mind?

Todo:
* [x] Unit tests
* [x] Code review (paging @ramirezg)

### CODE REVIEW

**Comment by batemapf** [comment]
:+1:

**Comment by mbland** [comment]
Just for the sake of mentioning prior art, [I've got a command-line script in a personal repo that does this](https://github.com/mbland/mbland-18f-utils/tree/master/tock). The `README.md` contains some sample output. I use it to grab the figures I need for my burndown spreadsheet. (Just added the per-project total this morning.)

**Comment by jmcarp** [comment]
Yep, the thought was to embed your tables in the Tock UI.

File: tock/projects/templatetags/project_tags.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+from django import template
+
+register = template.Library()
+
+
+def get(value, key):
+    return value.get(key)
+
+
+def default(value, default):
```
Comment (ramirezg) [inline tock/projects/templatetags/project_tags.py]: This is random but looks like a default function was [working](https://github.com/jmcarp/tock/blob/project-data-table/tock/tock/templates/projects/project_detail.html#L7) before you added this.

File: tock/tock/templates/projects/project_detail.html [hunk 2]
```diff
@@ -18,4 +19,35 @@ <h3 class="chart__title">User Hours Over Time</h3>
...
+      {% endfor %}
+    </tr>
+    {% endfor %}
+  </tbody>
+</table>
+
+<link rel="stylesheet" href="//cdn.datatables.net/1.10.7/css/jquery.dataTables.min.css" />
+<script src="//cdn.datatables.net/1.10.7/js/jquery.dataTables.min.js"></script>
+<script type="text/javascript">
+$(document).ready(function() {
```
Comment (ramirezg) [inline tock/tock/templates/projects/project_detail.html]: How long will this table get? Could the paging be turned on after 3 or 4 reporting periods?

File: tock/projects/templatetags/project_tags.py (hunk 1 above)
Comment (jmcarp) [inline tock/projects/templatetags/project_tags.py]: Good catch, will fix.

**Comment by jmcarp** [comment]
Now with unit tests.

@ramirezg: When there are lots of reporting period columns in the table, the table will scroll horizontally, and the user column will be fixed on the left side of the table, courtesy of the FixedColumns plugin for DataTables. There's definitely room for improvement, but I think this looks all right for now.

**Comment by ramirezg** [comment]
:+1:

**Pull request merged**